addons/repair/models/stock_picking.py

```diff
@@ -180,7 +180,8 @@ def _get_aggregated_records_by_date(self):
         # Make sure that all picking type IDs are represented, even if empty
         picking_type_id_to_dates = {i: [] for i in repair_picking_types.ids}
         picking_type_id_to_dates.update({r[0].id: r[1] for r in repair_records})
-        repair_records = [(i, d, _('Confirmed')) for i, d in picking_type_id_to_dates.items()]
```

Comment:
@KangOl can't you use `self.env._()`? As of v18: https://github.com/odoo/odoo/pull/174844

Reply:
Seems better.